### DISCUSSION TITLE
language

### DISCUSSION THREAD

**Discussion by ghost** [community/community#120947] [New to GitHub]
### Select Topic Area

Question

### Body

How to change the language of github ?

**Comment by davevad93** [comment]
Hi @Linux-Scratcher , since 2016 GitHub stopped supporting languages other than English for its interface, but you can change your trending settings to your preferred language by following these steps: 
- Click on your profile at the top right.
- In the drop down menu click on "Settings".
- Click on "Public Profile". 
- Scroll down until you see "Trending settings".
- Click on "Preferred spoken language" and choose your preferred language.
- Click on "Save Trending settings".

**Reply by ghost** [reply]
thanks

**Reply by davevad93** [reply]
You are welcome @Linux-Scratcher , if your question has been properly answered, please mark it as `answered`.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Linux-Scratcher! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by ghost** [reply]
okay thanks
